IndicatorFields/incidentfield-Cymulate_Immediate_Threats_Vector.json

```diff
@@ -0,0 +1,40 @@
+{
+	"associatedToAll": false,
+	"associatedTypes": [
+		"Cymulate Immediate Threats"
+	],
+	"breachScript": "",
+	"caseInsensitive": true,
+	"cliName": "cymulateimmediatethreatsvector",
+	"closeForm": false,
+	"columns": null,
+	"content": true,
+	"defaultRows": null,
+	"description": "",
+	"editForm": true,
+	"fieldCalcScript": "",
+	"group": 0,
+	"hidden": false,
+	"id": "incident_cymulateimmediatethreatsvector",
+	"isReadOnly": false,
+	"locked": false,
+	"name": "Cymulate Immediate Threats Vector",
+	"neverSetAsRequired": false,
+	"ownerOnly": false,
+	"placeholder": "",
+	"required": false,
```

Comment:
for all the json files add `"fromVersion": "5.0.0"` field

Reply:
all the incident types and incident fields,

Reply:
added